src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -5,4 +5,10 @@ public class ElevatorConstants {
   public static final double GEAR_RATIO = 0.0;
   public static final double MIN_HEIGHT = 0.0;
   public static final double MAX_HEIGHT = 0.0;
+
+  public static final double INTAKE_POSITION = 0.0;
```

Comment:
this one is kinda silly, but usually in the past we have set unknown constant values to 0-9 cause it makes it more clear -- there's probably a better way to do this but its tradition

Reply:
i somewhat dislike this tradition bc it can make shit not work

Reply:
lol yes but that is kinda the point -- if you don't have a value for it, you don't want it to work.

Reply:
hmmmmmmmmmmmmmmmmm

Reply:
ok sure